sdk/servicebus/Microsoft.Azure.ServiceBus/src/SessionHandlerOptions.cs

```diff
@@ -53,7 +53,7 @@ public TimeSpan MaxAutoRenewDuration
             }
         }
 
-        /// <summary>Gets or sets the time to wait for receiving a message.</summary>
+        /// <summary>Gets or sets the time to wait for receiving a message. This is the timeout the session is going to wait for the next available message before closing. This value has an impact on the message throughput.</summary>
```

Comment:
>This is the timeout the session is going to wait for the next available message before closing.  [](start = 76, length = 96)

This sounds good. Is it possible to expand this to be a bit more descriptive? Either here, or around the `RegisterSessionHandler` maybe we can mention how all the properties come together behind the scenes